[IMP] Website: add alt tag and metadata translation tips (#15180)
odoo/documentation: content/applications/websites/website/configuration/translate.rst

Translations
============

The website can be translated into multiple languages to allow visitors to view the content in their

> preferred language.

Reviewer: ```suggestion
Your website can be translated into multiple languages, allowing visitors to view its content in their
preferred language.
```
As discussed, you can use the "you" if it makes sense. In the intro paragraph, it often feels more natural to use "you can do X", so I would say "your website".

To make the content available in other languages, install the required languages and add them to the
website. Once this is done, a :ref:`Language selector <translate/language-selector>` is
automatically added to the website, allowing visitors to choose their preferred language.

Install languages

> =================

Reviewer: ```suggestion
Configuration
=============
```
I would simply use "configuration" here, making it clear it's the required configuration step before translating.

:guilabel:`Install new languages`. In the dialog box, select the :guilabel:`Languages`
from the dropdown menu, click :guilabel:`Add`, and click :guilabel:`Close`.

Then, add the language to the website by clicking on the :guilabel:`Languages` field, selecting the
desired language and clicking :guilabel:`Save`.To remove a language from the website, click the
:icon:`oi-close` :guilabel:`(close)` next to it.

.. _translate/default-language:

If the visitor’s browser language is not available, the website will display its content in the
database’s :ref:`default language <translate/default-language>`. To edit the default language, go to
:menuselection:`Website --> Configuration --> Settings`. Under the :guilabel:`General` section,
select the desired language in the :guilabel:`Default` field.

.. _translate/language-selector:

Language selector
=================

Once other languages are installed and added to the website, a :guilabel:`Language selector`
appears on the website, allowing the switch between languages. It automatically appears in
the header as a dropdown menu and in the :guilabel:`Copyright` section at the very bottom of the

> page.

Reviewer: ```suggestion
Once other languages are added to the website, a language selector is displayed in the website's header and footer, allowing visitors to switch between languages.
```
- I believe it can be shortened/simplified. Specifying it's in the footer's "copyright" section isn't really useful at this stage, for example, even if it's right.
- No :guilabel: for language selector (it's a bit confusing when to use guilabels or not, but in this case, we refer to the "concept" of language selector, not to a field/element where the name "language selector" would be visible on screen.

.. note::
   - The :guilabel:`Edit` button opens the translation mode only while viewing the page in a language
     that is not the default one. In this mode, only translations can be edited. Any other type of
     modification with the website builder must be carried out while using the default language.
     Any changes made to the default language pages are automatically applied to the pages of all
     other languages.

   - When a website supports multiple languages, the core URL structure remains consistent across
     languages, while specific elements like product names or categories are translated. For example,
     `https://www.mywebsite.com/shop/product/my-product-1` is the English version of a product page,
     while `https://www.mywebsite.com/fr/shop/product/mon-produit-1` is the French version of the same
     page. The structure (/shop/product/) stays unchanged, but the elements (e.g., product
     name) are translated to the selected language.

.. tip::
   Once the desired language is installed, some elements can also be translated from the backend
   (e.g., the :ref:`Translation <ecommerce/products/translation>` of a product name in the

>    product form).

Reviewer: ```suggestion
   Some elements can also be translated from the backend (e.g., product names).
```
To keep it simple.

Translation for SEO
-------------------

Once the pages are translated, translate the :ref:`alt tags <seo/alt-tag>` and
:ref:`meta tags <seo/meta-tags>` to help the

> :ref:`Search Engine Optimization <seo/content-optimization>`.

Reviewer: ```suggestion
```
Not really necessary, you can include the refs below instead.

> To translate the alt tags, switch to the language to translate,

Reviewer: ```suggestion
To translate an image's alt tag (i.e., the :guilabel:`Description` field) and title tag (i.e., the :guilabel:`Tooltip` field), switch to the language to translate,
```
I would clarify here, and add the info about the title tag.
